bin/metric-postgres-connections.rb

```diff
@@ -60,10 +62,11 @@ class PostgresStatsDBMetrics < Sensu::Plugin::Metric::CLI::Graphite
          short: '-P PORT',
          long: '--port PORT'
 
-  option :database,
-         description: 'Database name',
+  option :databases,
+         description: 'Database names, separated by ";"',
```

Comment:
I am thinking it might be safer to pass it with a comma separated list? Is it because we are passing this as raw SQL?

Reply:
You're right, using "," as separator is better than ";".
Also it was not escaped when used in a query.

I've updated the PR with patches for both.